stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetFlowController.kt

```diff
@@ -1,43 +1,46 @@
 package com.stripe.android.paymentsheet
 
-internal class PaymentSheetFlowController private constructor(
-    private val args: PaymentSheetActivityStarter.Args
-) {
+import android.content.Context
+import androidx.activity.ComponentActivity
+import com.stripe.android.paymentsheet.model.PaymentOption
 
-    internal companion object {
+internal interface PaymentSheetFlowController {
+
+    fun presentPaymentOptions(
+        activity: ComponentActivity,
+        onComplete: (PaymentOption?) -> Unit
+    )
+
+    fun confirmPayment(
+        activity: ComponentActivity,
+        onComplete: (PaymentSheet.CompletionStatus) -> Unit
```

Comment:
I think we landed on `PaymentResult` instead of `CompletionStatus`? Rename if so